packages/assets-controllers/package.json

```diff
@@ -96,11 +96,11 @@
     "typescript": "~5.2.2"
   },
   "peerDependencies": {
-    "@metamask/accounts-controller": "^19.0.0",
```

Comment:
We should include `assets-controllers` in this release. Otherwise, clients that use both `accounts-controller` and `assets-controllers` and upgrade to `accounts-controllers` 20 will break the existing peer dependency that `assets-controllers` has (which is `accounts-controller` 19).

Reply:
Yup, you are right! I've added it here: https://github.com/MetaMask/core/pull/4956/commits/00683a543a2b0ef59f4e7a42f9c810d4b6d758a1